dashboard/app/helpers/levels_helper.rb

```diff
@@ -74,6 +74,8 @@ def set_channel
   end
 
   def select_and_track_autoplay_video
+    return if @level.try(:never_autoplay_video) == "true"
```

Comment:
You could write a unit test for this and answer all of these questions :)

It would also prevent someone from breaking this code in the future if they
decide that serialized properties should start returning true/false instead
of "true"/"false"

On Tue, Sep 1, 2015 at 2:18 PM, Laurel Fan <laurel@code.org> wrote:

> what do you get when it's false? The string "false" is truthy in ruby so
> if you get that then you need to do this.
>
> On Tue, Sep 1, 2015 at 2:12 PM, Brad Buchanan <notifications@github.com>
> wrote:
>
>> In dashboard/app/helpers/levels_helper.rb
>> <https://github.com/code-dot-org/code-dot-org/pull/3819#discussion_r38472865>
>> :
>>
>> > @@ -74,6 +74,8 @@ def set_channel
>> >    end
>> >
>> >    def select_and_track_autoplay_video
>> > +    return if @level.try(:never_autoplay_video) == "true"
>>
>> Yeah, surprised me too. I think it's got to do with it being a serialized
>> attribute - maybe not necessary for this change?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/code-dot-org/code-dot-org/pull/3819/files#r38472865>.
>>
>
>